_workinggroups/dataanalysis.md

```diff
@@ -24,9 +24,9 @@ well-defined set of problems.
 [Meetings](https://indico.cern.ch/category/10914)
 [Forum](https://groups.google.com/forum/#!forum/hsf-analysis-wg)
 
-Convenors: Chris Burr (LHCb), Allison Hall (CMS), Paul Laycock (Belle II), TJ Khoo (ATLAS)
+Convenors: Chris Burr (LHCb), Allison Hall (CMS), TJ Khoo (ATLAS)
 
 ---
 
 Former Convenors:
-- Danilo Piparo (ROOT), Andrea Rizzi (CMS)
+- Danilo Piparo (ROOT), Andrea Rizzi (CMS), Paul Laycock (Belle II),
```

Comment:
```suggestion
- Danilo Piparo (ROOT), Andrea Rizzi (CMS), Paul Laycock (Belle II)
```

Reply:
Aside question: is that too much of a pain to add time frames to convenerships as we did for PyHEP, see the bottom of https://hepsoftwarefoundation.org/workinggroups/pyhep.html?